data/quarg/quarg missions.txt

```diff
@@ -259,3 +259,47 @@ mission "Ask Quarg About Coalition Early"
 			label end
 			`	Before you can ask it anything else, the Quarg bids you farewell, and walks away.`
 				decline
+
+
+
+mission "Quarg Pug Arfecta Warning"
+	minor
+	landing
+	to offer
+		has "First Contact: Quarg: offered"
+		has "flagship model: Pug Arfecta"
+	source
+		attributes "quarg"
+		attributes "ringworld"
+	on offer
+		conversation
+			`As soon as you land your ship a crowd of visibly irritated Quarg start to gather, and as you go to disembark you find a rather imposing Quarg blocks your exit while staring piercingly at your ship.`
+			`	"It is as though a baby in swaddling has tamed a dragon and now rides it through the sky spewing fire with reckless abandon," it says with a mix of amusement and disgust.`
+			choice
+				`	"It's a long story..."`
+					goto evasive
+				`	"It was an act of righteous defiance."`
+					goto righteous
+				`	"The Pug fancy themselves gods, but since God is dead..."`
+					goto anarchy
+				`	"I saw something I liked, so I made it mine."`
+					goto greedy
+			label evasive
+			`	It scoffs, "It matters not how it happened."`
+				goto end
+			label righteous
+			`	“You fancy yourself a savior or a revolutionary then?” the Quarg says with what you assume is a sarcastic tone.`
+				goto end
+			label anarchy
+			`	"So you took it upon yourself to prove them wrong?" the Quarg says with an amused, incredulous smile.`
+				goto end
+			label greedy
+			`	"A baby in swaddling, as I said before."`
+				goto end
+			label end
+			`	"We give you a warning. You know not what you have child, the weapon you wield."`
+			`	"We have seen its power. We have felt its power. We have learned from its power. But you and your kind know nothing of this power and its burdens and dangers. Be careful that you direct this power, and not it you."`
+			`	"And be careful to remember that we have learned to overcome it."`
+			`	The Quarg curtly motions farewell then takes one last long look at your ship before leaving you to your business. The crowd too starts to disperse, with a few staying behind to gawk at your ship.`
+			`	As you make your way through the ring you notice all the Quarg onboard glaring at you with disgust.`
```

Comment:
```suggestion
			`	As you make your way through the ring you notice many of the Quarg onboard glaring at you with disgust.`
```
This feels like a stronger response than is set up by the rest of the conversation, especially considering the previous line has the stragglers of the crowd "gawking" at your ship in awe rather than regarding you with disgust. Is the crowd composed only of non-Quarg?

Reply:
Makes sense will update directly because of a line conflict

Reply:
Change made